freesound/settings.py

```diff
@@ -530,6 +530,17 @@
 MAX_FILESIZE_FOR_ANALYSIS = 5 * 1024 * 1024 * 25
 
 
+# -------------------------------------------------------------------------------
+# Rate limiting
+
+RATELIMIT_SEARCH_GROUP = 'search'
+RATELIMIT_SIMILARITY_GROUP = 'similarity'
+RATELIMITS = {
+    RATELIMIT_SEARCH_GROUP: '2/s',
+    RATELIMIT_SIMILARITY_GROUP: '2/s'
+}
+BLOCKED_IPS = []
```

Comment:
this will require a deploy each time we need to change an ip address.
As we're not in an urgent rush right at the moment, I'd like to adapt it to allow it to read these values from a cache.

We should probably not make it read every single request, as that would make 1 more redis/memcache read on every single request. Instead we could save a global variable along with a "last updated" flag, and if the last updated is greater than some time (5 mins?) then we do a cache call to update the list.